documents/meetup-starter-kit.md

```diff
@@ -0,0 +1,82 @@
+* "Co-working spaces have been our go-to meetup venue hosts, but with attendance sometimes surpassing 100 people, it's a challenge to find a reliable, free venue." @danmactough on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145729309)
+* "We try to put on themed nights such as IoT, performance, testing but this takes a LOT of work to approach speakers and maybe a bit beyond beginner." @iancrowther on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145829320)
+* "We also run a segment called Community Announcements which is great as its very interactive with the audience. [...] Getting the audience on the mic makes networking super easy and ensures people know who to speak to and don't miss out. Everyone has had a few beers by then so its a lot of fun, some heckling and general mayhem. As a host, this is my favorite part of the event as I honestly have no ideas what's coming." @iancrowther on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145829320)
+
+
+## RSVP
+Using an online event/meetup planning site is the standard. [Meetup.com](http://www.meetup.com/) and
+[ti.to](https://ti.to/) are two that are used at a few popular Node meetups. You can base
+the number of RSVPs available on your venue's capacity.
+
+**Food and drink** reserves are needed for the event. RSVPs are important for
+measuring food and drink coverage, as they help you gauge how much you need.
+
+#### Protips:
+* "We base the number of RSVPs depending on the capacity of the space we're using for the month." @paulgrock on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145303436)
+* "We use meetup.com to manage RSVPs so we can limit attendance to the venue capacity and plan food/beverage. We typically see ~50% of RSVPs actually attend." @danmactough on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145729309)
+* "We do a few things here, we have provided block tickets to some code academies meaning they just have 10 tickets a month to distribute as they wish. Its keeps the door open to new attendees from new semesters etc. These targeted block ticket allocations allow us to shape the audience and keep it a mix of beginner through to advanced and allows us to be a very diverse crowd.. not just the developer stereotype." @iancrowther on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145829320)
+
+
+## Sponsors
+Sponsors can help cover costs of the event. Food and drink, venue, gear, and
```

Comment:
Maybe a note here like "sponsors aren't always necessary." Taking on sponsors is actually a lot of extra work so it might encourage more meetups if we let people know that they aren't strictly necessary.